athena/MapData.tsx

```diff
@@ -417,7 +416,8 @@ export default class MapData {
     value: T,
   ): T {
     const { map, size } = this;
-    for (let i = 0; i < map.length; i++) {
+    const len = map.length;
```

Comment:
Can you rename all of these to `length`? The codebase is trying to avoid abbreviations as much as possible.